concepticondata/conceptlists.tsv

```diff
@@ -305,3 +305,5 @@ Smeltzer-1995-300	Smeltzer, Brad and Smeltzer, Susan	1995		300	basic, questionna
 Rzymski-2020-1624	Christoph Rzymski, Tiago Tresoldi, Simon J. Greenhill, Mei-Shin Wu, Nathanael E. Schweikhard, Maria Koptjevskaja-Tamm, Volker Gast, Timotheus A. Bodt, Abbie Hantgan, Gereon A. Kaiping, Sophie Chang, Yunfan Lai, Natalia Morozova, Heini Arjava, Nataliia Hübler, Ezequiel Koile, Steve Pepper, Mariann Proos, Briana Van Epps, Ingrid Blanco, Carolin Hundt, Sergei Monakhov, Kristina Pianykh, Sallona Ramesh, Russell D. Gray, Robert Forkel and Johann-Mattis List	2020		1624	annotated	English	Global	http://clics.clld.org	Rzymski2020		This is the concept list underlying the third release of the [Database of Cross-Linguistic Colexifications (CLICS3)](http://clics.clld.org) (for the first version see, [List 2014](:bib:List2014). The data comprises 3156 language varieties and 2906 distinct Concepticon concept sets. Instead of providing full graphs for the networks, we provide links to the major clusters for each concept in CLICS. The column ``CENTRAL_CONCEPT`` lists the most central node in each of the community clusters in CLICS2. The community identifier is listed in column ``COMMUNITY``. The columns ``FAMILY_FREQUENCY``, ``LANGUAGE_FREQUENCY`` and ``WORD_FREQUENCY`` list the number of times a translation for a given concept occurs in the CLICS2 data. The columns ``DEGREE``, ``WEIGHTED_FAMILY_DEGREE``  and ``WEIGHTED_LANGUAGE_DEGREE`` contain the unweighted and weighted degree for each node.		
 Morse-1996-375	Morse, Nancy L.	1996		375	areal, questionnaire	Spanish, English	Sáliba	https://www.sil.org/resources/archives/69059	Morse1996		This list of 375 items was used as a questionnaire for the indigenous South American language Sáliba. The items to be elicited were given both in English and Spanish. The author marked some items with asterisks that were taken from Swadesh-Rowe lists, most likely the [Swadesh 1955 100-item](:ref:Swadesh-1955-100) list, although a few of these marked items did not match any Swadesh list.		
+Zettersten-2020-24a	Zettersten, Martin and Lupyan, Gary	2020		24	naming test, colors	English	English	https://doi.org/10.1016/j.cognition.2019.104135	Zettersten2020		This list comprises the color stimuli used in experiment 1A, 1B, 2A, 2B in [Zettersten and Lupyan (2020)](:bib:Zettersten2020). The study conducted seven experiments in which participants learned categories composed of colors and shapes that were either easy or more difficult to name in English.		
+Zettersten-2020-16b	Zettersten, Martin and Lupyan, Gary	2020		16	naming test	English	English	https://doi.org/10.1016/j.cognition.2019.104135	Zettersten2020		This list comprises the shape stimuli used in experiment 3A and 3B in [Zettersten and Lupyan (2020)](:bib:Zettersten2020). The study conducted seven experiments in which participants learned categories composed of colors and shapes that were either easy or more difficult to name in English. The association values come from [Vanderplas and Garvin (1959)](:bib:Vanderplas1959).		
```

Comment:
Missing the `a` and `b` suffix in the `LIST_SUFFIX` columns

Reply:
Thanks! I added them accordingly.